biomage/stage/stage.py

```diff
@@ -204,6 +205,10 @@ def get_sandbox_id(templates, manifests, all_experiments):
         ]
     )
 
+    # if we are in auto mode (non interactive) just generate a random sandbox ID name
+    if auto:
+        return random_name.generate_name()
```

Comment:
Just a question, why not just use the default name generated below?

Reply:
1. because I prefer easy to remember names.
2. because the generator will fail if it ends up with `-` and between fixing that and getting random names in auto, I preferred the second one.